packages/teams-js/src/private/interfaces.ts

```diff
@@ -271,3 +272,60 @@ export interface UserJoinedTeamsInformation {
    */
   userJoinedTeams: TeamInformation[];
 }
+
+// externalAppCardActionsInterfaces.ts
+
```

Comment:
All exported types in here should have comments describing what they are for, and they should be marked as @hidden and @internal if they are exported out of the library for app developers